python/perf-kernels/flash-attention.py

```diff
@@ -271,18 +285,7 @@ def _attn_fwd_inner(acc, l_i, m_i, q, k_ptrs, v_ptrs, bias_ptrs, stride_kn, stri
                 size_n = start_n + OFFS_N[None, :]
                 mask = size_n < boundary_m[:, None]
                 qk = tl.where(mask, qk, float("-inf"))
```

Comment:
Moving MASK_STEPS after tl.dot makes the test pass also without requiring extra conversions